docs/static/security/logstash.asciidoc

```diff
@@ -49,7 +49,7 @@ POST _security/role/logstash_writer
   "cluster": ["manage_index_templates", "monitor", "manage_ilm"], <1>
   "indices": [
     {
-      "names": [ "logstash-*" ], <2>
```

Comment:
Thanks for your contribution! We're working through the implications to be sure that this fix works for all users, and we're also considering the impact of ECS on by default.